lighthouse-core/audits/accessibility/layout-table.js

```diff
@@ -12,6 +12,23 @@
  */
 
 const AxeAudit = require('./axe-audit');
+const i18n = require('../../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of an accesibility audit that evaluates if a table used for layout contains any data elements such as table headers as it can be confusing for screen readers. This descriptive title is shown to users when no user action is required. */
```

Comment:
Might need to explain what presentational means moreso in this context?  Or switch to a less complex word or phrasing.

Reply:
Added a bit more detail 👍 `presentation` is a hard requirement of the `aria-role`